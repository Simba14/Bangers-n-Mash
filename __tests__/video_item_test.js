import React from 'react';
import VideoItem from '../src/components/video_item';
import renderer from 'react-test-renderer';

test('renders the VideoItem component correctly', () => {
  const props = {
    video: { default: 'id' }
  };

  const tree = renderer.create(
    <VideoItem {...props}/>
  ).toJSON();
  expect(tree).toMatchSnapshot()
});
