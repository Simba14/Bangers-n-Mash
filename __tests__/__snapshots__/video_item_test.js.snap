// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the VideoItem component correctly 1`] = `
<div
  className="col-md-6 video"
>
  <div
    className="embed-responsive embed-responsive-16by9"
  >
    <iframe
      className="embed-responsive"
      src="https://www.youtube.com/embed/undefined"
    />
  </div>
  <div
    className="video-info"
  >
    <div
      className="video-title"
    />
    <div />
    <div
      className="video-description"
    />
    <strong
      className="tracklist-heading"
    >
      Tracklist:
    </strong>
    <div
      className="video-tracklist"
    />
  </div>
</div>
`;
